apps/web/pages/apps/[slug]/setup.tsx

```diff
@@ -36,11 +36,4 @@ export default function SetupInformation(props: InferGetStaticPropsType<typeof g
 
 SetupInformation.PageWrapper = PageWrapper;
 
-export const getStaticPaths: GetStaticPaths = async () => {
-  return {
-    paths: [],
-    fallback: "blocking",
-  };
-};
-
-export { getStaticProps };
+export { getServerSideProps };
```

Comment:
What's the motivation for this change?

Reply:
Got it. Alby needs to do SSR which is a first I guess.

Reply:
The Alby setup page needs to retrieve app credentials to check if the app has been setup or not.

Possibly it could be done with trpc instead? But, I am wondering do the setup pages make sense to use getStaticProps in the first place if they are for installed apps that require a logged in user?